chainer/variable.py

```diff
@@ -1303,14 +1309,10 @@ def addgrad(self, var):
             self.initialize(var.shape)
 
         dst = self.grad_var
-
-        src_dev = cuda.get_device_from_array(src.array)
-        dst_dev = cuda.get_device_from_array(self.array)
-
-        if src_dev.id != dst_dev.id:
-            src = chainer.functions.copy(src, dst_dev.id)
-        self._grad_var = src if dst is None else src + dst
-        self._grad = None
```

Comment:
Could you explain why is this line removed?

Reply:
`_grad` is set inside `grad_var.setter`.
(I don't know why previously `None` was always assigned).

Reply:
I understand. Thanks.